string name
uint32 timeouts
uint32 checksum_errors
